### ISSUE TITLE
nx-deploy-it could not be installed

### ISSUE THREAD

**Issue by vdiatsigkos** [Dev-Thought/nx-plugins#79]
**Describe the bug**
When I am trying to install nx-deploy-it package fails with error `Failed at the @dev-thought/pulumi-npm@1.6.4 postinstall script`.

**To Reproduce**
run `npx nx add @dev-thought/nx-deploy-it`

**Logs**
```
npm ERR! code ELIFECYCLE
npm ERR! errno 40
npm ERR! @dev-thought/pulumi-npm@1.6.4 postinstall: `node src/install-pulumi`
npm ERR! Exit status 40
npm ERR!
npm ERR! Failed at the @dev-thought/pulumi-npm@1.6.4 postinstall script.
```

**Application type:**
[x] Angular
[x] NestJS

**Provider:**
[] AWS
[x] Azure
[] Google Cloud Platform

**nx-deploy-it version**
`2.0.0`

**Comment by MitkoTschimev** [comment]
Which operating system are you using?

**Comment by diatsigkosv** [comment]
> Which operating system are you using?

Windows 10

**Comment by aterreno** [comment]
Hey @MitkoTschimev I just tried `npx nx add @dev-thought/nx-deploy-it` on mac and it doesn't work as well. 

is this project still in active development?

**Comment by MitkoTschimev** [comment]
Yes it is but I don't have so much time. I used it just 2 days ago for a small side project and I had an issue with missing angular devkit. That happens because I still have an open branch where I'm migrating to the new nx devkit completely. 

Can you check if you had the same issue with angular devkit?

**Comment by aterreno** [comment]
@MitkoTschimev  I don't use angular

**Comment by MitkoTschimev** [comment]
Sure but nx was built on top of angular schematics. Now everything is built on top of their own nx devtools. But all the plug-in providers need to migrate ;)

What kind of error do you get?

**Comment by aterreno** [comment]
❯ yarn add -D @dev-thought/nx-deploy-it
yarn add v1.22.10
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
error /Users/toni/code/aterreno/pulumi-recipies/node_modules/@dev-thought/pulumi-npm: Command failed.
Exit code: 10
Command: node src/install-pulumi
Arguments:
Directory: /Users/toni/code/aterreno/pulumi-recipies/node_modules/@dev-thought/pulumi-npm
Output:
Unfortunately, your platform and architecture (darwin, arm64) isn't currently supported by Pulumi. Please uninstall this package.

It's because of my M1 I guess. But Pulumi is fine now with this

**Comment by aterreno** [comment]
I see the issue comes from upstream from https://github.com/dev-thought/pulumi-npm

I might give it a go and try to fix it

**Comment by MitkoTschimev** [comment]
Ah ok, that's possible. Unfortunately, I don't have the M1 so maybe you can try to fix it in the package pulumi-npm. Would be a great contribution.

**Comment by aterreno** [comment]
Yep I've done that. Check the pr in that project.

**Comment by MitkoTschimev** [comment]
Released version 1.7.0
